core/src/it/scala/cr/pulsar/PulsarSpec.scala

```diff
@@ -112,6 +116,59 @@ class PulsarSpec extends PulsarSuite {
       }
     }
 
+    test(
+      "A consumer fails to decode first message, which is acked and logged via onFailure, and continue to process more messages"
+    ) {
+      (
+        Deferred[IO, Either[Throwable, Unit]],
+        Ref.of[IO, Option[Event]](None),
+        Ref.of[IO, Option[Consumer.DecodingFailure]](None)
+      ).tupled
+        .flatMap {
+          case (latch, ref, logger) =>
+            val nkTopic = topic("decoding-failure-acked-logged")
```

Comment:
Nitpicking: `mkTopic`

Reply:
nk was correct, it was meant for `nacked` but that changed and forgot to update it.